[doc] Add the spelling checker in the install guide (#7193)
PyCQA/pylint: doc/user_guide/installation/with-multiple-interpreters.rst

.. _continuous-integration:

Installation with multiple interpreters
=======================================

> Of course it's always possible and safer to analyse on each interpreters.

Reviewer: Should this be on a new page? I think it fits well in the previous place but I just don't think we should recommend to use the latest interpreter. That's really something that's up to the user's discretion.

Author: Hmm well pylint does work better with the latest interpreter because the ast get better all the time. Python 3.5 can't understand f-string, but 3.6 does. So sure the user choose their interpreter but we need to warn them of the consequences.

> It's always possible and safer to check on each interpreters.
> 
> But, it's also possible to analyse code written for older interpreters by using

Reviewer: ```suggestion
It's possible to analyse code written for older or multiple interpreters by using
```

> the ``py-version`` option and setting it to the old interpreter. For example you can check

Reviewer: ```suggestion
the ``py-version`` option and setting it to the oldest supported interpreter of your code. For example you can check
```

that there are no ``f-strings`` in Python 3.5 code using Python 3.8 with an up-to-date

> pylint even if Python 3.5 is past end of life (EOL) and the latest pylint are not

Reviewer: ```suggestion
pylint even if Python 3.5 is past end of life (EOL) and the version of pylint you use is not
```

compatible with it.

> We do not guarantee that ``py-version`` will work for all EOL python interpreters indefinitely,

Reviewer: ```suggestion
We do not guarantee that ``py-version`` will work for all EOL Python interpreters indefinitely,
```

> (for anything before python 3.5, it probably won't). If a newer version does not work for you,

Reviewer: ```suggestion
(for anything before Python 3.5, it probably won't). If a newer version does not work for you,
```
